gobblin-runtime/src/main/java/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -331,6 +321,7 @@ protected void postProcessTaskStates(List<TaskState> taskStates) {
    * Subclasses can override this method to do whatever processing on the {@link JobState} and its
    * associated {@link TaskState}s, e.g., aggregate task-level metrics into job-level metrics.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
No, this is not deprecated. `postProcessTaskStates` is. This is just to surpress deprecation warning because of the call to `postProcessTaskStates`.